### PR TITLE
[chore] Ignore built pipelines

### PR DESCRIPTION
**Pull request by Bobgy**
I prefer ignoring built pipelines so they don't pollute `git status` output.

This change is [Reviewable](https://reviewable.io/reviews/kubeflow/pipelines/2638)

### CODE REVIEW

**Comment by Bobgy** [comment]
@IronPan @Ark-kun I assume you have some context with this. What do you think?

File: .gitignore [hunk 1]
```diff
@@ -66,3 +66,5 @@ _artifacts
 # Generated Python SDK documentation
 docs/_build
 
+# Pipeline Built Outputs
+samples/**/*.tar.gz
```
Comment (Ark-kun) [inline .gitignore]: +`samples/**/*.zip` probably

**Comment by Bobgy** [comment]
Note the failure may be caused by node version difference, you can try to change https://github.com/kubeflow/pipelines/issues/2386

**Comment by Bobgy** [comment]
I realized paramterized_tfx_oss commited its compiled pipeline in the repo, so we cannot add this.
/close

**Pull request closed without merging**